prog/net-plugin/launch.sh

```diff
@@ -46,6 +46,7 @@ exec /home/weave/weaver $EXTRA_ARGS --port=6783 $(router_bridge_opts) \
     --host-root=/host \
     --proc-path=/host/proc \
     --http-addr=$HTTP_ADDR --status-addr=$STATUS_ADDR \
+    --without-proxy \
```

Comment:
Done in 974b867dd672e024ad9527dbcaea4c82d3608736 `Add --proxy option to enable/disable proxy.`